packages/file-explorer/src/FileStorage.ts

```diff
@@ -11,7 +11,7 @@ export interface FileMetadata {
   basename: string;
 }
 
-export type FileType = 'file' | 'directory';
+export type FileType = string;
```

Comment:
Instead of this change, we should throw in `GrpcFileStorage` if the type does not match a known type `'file' | 'directory'`